docs/how-tos/sql-user-management-using-postgres-users-and-passwords.rst

```diff
@@ -0,0 +1,337 @@
+.. _sql-user-management-using-postgres-users-and-passwords:
+
+SQL User Management using postgres' users and passwords
```

Comment:
You can avoid the [spellcheck error](https://github.com/PostgREST/postgrest-docs/actions/runs/3460198539/jobs/5776477261) by using `PostgreSQL's` instead since it's included in [our dict](https://github.com/PostgREST/postgrest-docs/blob/main/postgrest.dict#L124-L128).

Or you can also add `postgres` and `postgres'` there.

```suggestion
SQL User Management using PostgreSQL's users and passwords
```